### DISCUSSION TITLE
Account restricted for violation of terms of service?

### DISCUSSION THREAD

**Discussion by ValdemarusS** [codewars/codewars.com#2513] [Q&A] [answered]
Hello. Explain to me, please, why [my account](https://www.codewars.com/users/ValdemaruS) is blocked?

**Comment by Blind4Basics** [comment]
~~because you are a copycat.~~ (technical note: it's not blocked, it's "restricted")

**Reply by ValdemarusS** [reply]
What is copycat? In what kata did the robot think I did this? And how to get rid of this status if it is not true?

P.S. Excuse my machine speech. I don't speak English well, so I use google translate 😔

**Comment by Blind4Basics** [comment]
copycat = plagiarist

getting rid of this restriction: you'll have to convince me. You'll have a hard time, considering the stats on your completions.

**Reply by ValdemarusS** [reply]
I see no point in dumb copy and paste. I solve only those kata that arouse my personal interest, and not everything. If you look at my history, you can see some progress (albeit small), from simple to more complex, which is logical, I think. I often comment on my decisions. Before codewars, I spent some time solving problems on ProjectEuler. I do not know how else can I convince you otherwise?

**Comment by Blind4Basics** [comment]
do you have a github account or a discord one? If you are on discord, poke at me on CW's group (-> "B4B")

**Reply by musadiqmustafa** [reply]
my account is restricted and my honor decrease but  I don't know y
I didn't cheat

**Reply by Blind4Basics** [reply]
You are a plagiarist, that's the reason.
Hunting for solution on the web is not considered as a honorable way to earn... honor.

**Reply by musadiqmustafa** [reply]
I m not i was just use vs code to check the codes which i used to made from chunks

**Reply by musadiqmustafa** [reply]
Can u rechecked please

**Reply by Blind4Basics** [reply]
I already double checked. Being a plagiarist has nothing to do with using an IDE or not.

**Reply by VENDETTA2212** [reply]
My account is also restricted, what should I do? can you fix that?

**Comment by Blind4Basics** [comment, marked as answer]
Stiuation clarified. Restriction suppressed.

**Reply by ValdemarusS** [reply]
Thank you for understanding. It's important for me 😊

**Comment by Leydy** [comment]
I've been copying some other people's code but just to check my logic, I didn't know I was going to get banned, I need to get my honor back please :(

**Reply by Blind4Basics** [reply]
Hi,

And your account is...?

**Reply by Leydy** [reply]
Hi, my account is herzeleid 5kyu

**Reply by Blind4Basics** [reply]
oh. No, you definitely won't get those points back.

**Reply by Leydy** [reply]
Now that you have reset my account, can I see my progress with my clan?

**Reply by Blind4Basics** [reply]
what do you mean?

**Reply by Leydy** [reply]
anyway, thanks!

**Comment by hardikx26** [comment]
Hello my account is also restricted, please help @hardik26

**Comment by acordova200** [comment]


**Reply by Blind4Basics** [reply]
just delete it and start over. And this time, don't "cheat".

**Reply by HowToType** [reply]
Hey, can you unban my account https://www.codewars.com/users/HowToType%3F, I got restricted right after solving a 2kyu kata, like seconds after solving it, Idk if my code matched with someone else or something but I definitely didn't copy and paste it,

**Reply by Blind4Basics** [reply]
I didn't see this message earlier, sorry.

> I definitely didn't copy and paste it,

yeah, you translated it into another language instead. But that's still being a copycat. Also, your other stats point you as a copycat on too many kata.

Advise: delete the account and start fresh. Fairly, this time.

**Reply by Peter-developer01** [reply]
Can I also delete the account and start fresh? Without copypasting, of course.

**Reply by Blind4Basics** [reply]
Sure. And don't use alter accounts (your other account also got locked. I suggest you delete both)

**Reply by Peter-developer01** [reply]
Yeah, already deleted other account. Thanks.

**Comment by Arenesz** [comment]
Hello my Account on code wars were restricked too.  re rolled my account on the of the very exact email.  and it got restricted after that

**Reply by Blind4Basics** [reply]
Hello,

And do you have any actual question about that? (if so, what's your CW username?)

**Comment by hardik26** [comment]
Hi,
Request you to do not add me in this mail thread thanks

**Reply by Blind4Basics** [reply]
you can unfollow/untrack the thread, afaik, but only you can do that. You should have an unsubscription link in the mail.

**Comment by shrikharg** [comment]
Hi, the same has happened with my account - Shrikhar0973.
I believe the account got restricted right after I submitted a 2 kyu kata which was unfortunately not my own code. However, my situation demanded such an extreme step. I am a college student and my homework assignments are done on Codewars with the honour points having a direct bearing on my grade. On my behalf it is very wrong that I cheated, but it was my last option for a 100 point boost I required. All other katas have been meticulously done by me and most of them are 8-5 kyu katas.
I request you to remove the restriction on my account and if possible replenish my points excluding the 2 kyu's points as well as a penalty from your side.
I sincerely hope you understand my situation and offer your kind help.
Thank you

**Reply by Blind4Basics** [reply]
Hello,

Yes you can delete and start fresh. But this time, don't use copies or the result will be the very same.

That situation (your school requirements) is very unfortunate, to say the least... We already got several cases like this over the past few days/weeks and it would be good that CW admins could discuss the matter with your teacher(s). Don't hesitate to escalate the request on your side. The sooner the better, for everyone.

Cheers

**Reply by shrikharg** [reply]
Thanks for your response. Would I have to surely delete and restart? I have around 150 points after the restriction. Can I be allowed to continue instead please?

**Reply by Blind4Basics** [reply]
You also can keep the current one, yes, but it will stay locked (for now at least). That won't forbid you to solve kata, you just won't be able to do a lot of other things.

**Reply by shrikharg** [reply]
I am not able to see other solutions for completed katas currently. This hinders learning from my side in my humble opinion. Is there a possibility of me being able to enjoy previous privileges again?
Sorry for the inconvenience.

**Reply by Blind4Basics** [reply]
I understand the inconvenience, yes, but as long as the matter isn't sorted out with your teacher(s), I'd prefer to leave the restriction in place. Problem is: if I unlock yours, I'll suddenly see hundreds of other users saying "hey! I'm also from that school!". The situation would quickly become unmanageable (there are loads of restricted accounts each week).

**Reply by shrikharg** [reply]
I understand completely. Do you suggest deletion and restarting over continuing with the same account?
Hope I don't get restricted again as I'll have to make quick progress (without cheating of course) to catch up within the next few days.

**Reply by Blind4Basics** [reply]
That's up to you, you definitely can delete and start fresh.

Quick progress is ok as long as the solutions are original (note that stuff like changing a var name or function name here or there is not considered enough to make a solution "original")

**Reply by shrikharg** [reply]
Got it, thanks for your time and help! Until we talk again ;)

**Reply by Blind4Basics** [reply]
No problem. 
> Until we talk again ;)

I just hope it will be about your teacher getting in touch with our admins... ;o

**Reply by shrikharg** [reply]
I wish haha!

**Reply by shrikharg** [reply]
Hey,
As you had suggested, I deleted my restricted account and restarted with the same username - Shrikhar0973. I worked hard on predominantly 8-6 kyu problems by myself till I reached 500+ honour points.
Now however my account has been restricted for seemingly no reason. I had not used any unfair means to solve katas and now with a deadline within 2 days, have no points to show topped off with a restriction.
I feel helpless at this point.
Request you to perhaps look into it and help me out. I am extremely short of time.

**Reply by Kacarott** [reply]
I am not the one who invalidated your solutions, but looking through them many of them look very much like they were copy pasted from somewhere. That would be considered unfair means.

**Reply by Blind4Basics** [reply]
oh, lemme check again

**Reply by shrikharg** [reply]
I solve my programs on VS code and do not name my variables lazily. Also I
comment as it is an assessed project. I hope you can understand my case
here.

**Reply by Blind4Basics** [reply]
restoring. Sorry about that

**Reply by shrikharg** [reply]
I really can't thank you enough.

**Reply by Blind4Basics** [reply]
actually, there is a way you can: get your teacher in touch with the admins.

**Reply by Kacarott** [reply]
I think you are very lucky. I strongly suggest against using ChatGPT in the future if you want to keep your points.

**Reply by shrikharg** [reply]
Did it last time and paid the price for it sir. Not on the new account though.

**Reply by shrikharg** [reply]
@Blind4Basics too late almost, our course ends in a couple days ;(

**Reply by Blind4Basics** [reply]
Never too late, if that can lead them to not do the same mistake in the next years.

chatGPT: be aware that asking it to rewrite a solution is just the same...

**Reply by shrikharg** [reply]
okay, I will try. ChatGPT - yes I am aware, learnt it the hard way. That's why I am doing rather easy 8-5 kyu ones for which reference is not needed. Thanks again.

**Comment by mortega16** [comment]


**Reply by Blind4Basics** [reply]
You were using alter accounts to unlock solutions.

**Comment by simonbenii** [comment]


**Reply by Kacarott** [reply]
You want the points which you got by cheating, to be given back to you? If you need points to start an exam, then I suggest you start solving kata properly without cheating to actually *earn* the points, instead of trying to cheat your way into the exam.

**Comment by Kirpich158** [comment]
hello. after many years I logged into my old account and found out that I was restricted and can't do katas. idk the exact reason, but I was wondering, no matter the reason, is there any way to get back to normal status to do katas or it's just one-way restriction with no 2nd chance?

**Reply by hobovsky** [reply]
It is difficult to tell without knowing what account it was, because mods cannot check what exactly was the problem. But generally, the options are:
- delete the old account and start from scratch, this time possibly without doing the thing which led to the initial suspension.
- write an email to `info@codewars.com` with a link to your user profile, and ask for another review of the suspension. It might work if the offense was not excessive,  or might not if it involved many cheat solutions or copies of other solutions.
- ask in the `#contact-mods` channel of Codewars Discord.

**Reply by Kirpich158** [reply]
gotcha, thx for the info, will contact by email then
